Sources/Quick/World.swift

```diff
@@ -38,6 +38,11 @@ final internal class World: NSObject {
     */
 
     internal var currentExampleMetadata: ExampleMetadata?
+    
+    /**
+        The descriptor of the current context.
+    */
```

Comment:
Agreed, it doesn't provide any extra value. I left it in there in hopes that maybe someone would have a suggestion of how to better name it :) 

After marinating for a couple days, maybe I can propose something better. Otherwise the update will remove it.